libs/template/presets/storefront/experience/pages/checkout-page.ts

```diff
@@ -83,7 +83,7 @@ export const checkoutPage: StaticComponent = {
           type: 'oryx-content-text',
           content: {
             data: {
-              text: '<p>The <a href="/article/terms-and-conditions">Terms and conditions</a> apply.<br/>Please also see our <a href="/article/privacy">Privacy notice</a>.</p>',
+              text: '<p>The <a href="/article/terms-and-conditions" color="primary">Terms and conditions</a> apply.<br/>Please also see our <a href="/article/privacy" color="primary">Privacy notice</a>.</p>',
```

Comment:
I'm thinking how good it's to use standard html attribute for styling. For now we are using it to set the color only, which is looking kind of semantic, but at some point someone can add more styles to it. I'd probably change it to data attribute.